.snippets/code/infrastructure/general/bootnode.conf

```
@@ -0,0 +1,19 @@
+server {
+       listen       30312 ssl http2 default_server;
+       server_name  dot-bootnode.stakeworld.io;
+       root         /var/www/html;
+
+       ssl_certificate "<your_cert";
+       ssl_certificate_key "<your_key>";
```

```suggestion
       ssl_certificate "INSERT_YOUR_CERT";
       ssl_certificate_key "INSERT_YOUR_KEY";
```

placeholders should be capitalized and start with INSERT_ for consistency across the docs